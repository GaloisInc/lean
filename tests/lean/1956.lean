example : 1000 * 1000 = 123456 :=
rfl
