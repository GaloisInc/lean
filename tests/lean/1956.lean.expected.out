1956.lean:1:0: error: deep recursion was detected at 'replace' (potential solution: increase stack space in your system)
